test/utils/index.ts

```diff
@@ -151,7 +151,7 @@ export default async function (chain2?: boolean): Promise<PreTest> {
   const chainId: BytesLike = '0x' + network.holographId.toString(16).padStart(8, '0');
   const chainId2: BytesLike = '0x' + network2.holographId.toString(16).padStart(8, '0');
   const fixtures: string[] = [
-    'HolographGenesis',
+    ['localhost', 'localhost2'].includes(hre.networkName) ? 'HolographGenesisLocal' : 'HolographGenesis',
```

Comment:
do we need to do this HolographGenesisLocal?

Reply:
Deploy script fails when deploying locally, if the deployed contract is not the HolohraphGenesisLocal one, as the following if will fail to work: https://github.com/Think-and-Dev/holograph-protocol/blob/f1943fefc514a684ae8cec6ac967b9955db988bd/deploy/00_genesis.ts#L20

Reply:
its ok for me!